profilepage/profilepage.js

```diff
@@ -0,0 +1,122 @@
+
+$document.ready(function() {
+  validateProfilePage();
+  $(".form-textfield input").blur(function)) {
+    var isBlank = $.trim($(this).val().length > 0;
+    $label = $(this).siblings(".form-label").toggleClass("form-label__floating",isBlank);
+
+  });
+});
+
+function validateProfilePage() {
+  $("#vitaProfileEdit").validate({
+    rules: {
+      "firstNameProfile":"required",
+      "lastNameProfile":"required",
+      email: {
+        required:true,
+        email: true
+      },
+      phone: {
+        required:true;
+      }
+    },
+    messages: {
+      email: {
+        required: "We need your email to confirm your shift times",
+        email: "Your email must be in the form of name@example.com"
+      }
+    }
+  });
+}
+function taxFunction() {
```

Comment:
This is very poorly named. You really shouldn't call methods `xxxFunction`. We know it's a function, there's no reason to put that in the name. Also `taxFunction` tells me nothing about what this does. I honestly don't know what this function does, but maybe something like `showTaxOptions` is more appropriate?